src/main/treeDataProviders/utils/dependencyUtils.ts

```diff
@@ -541,6 +540,18 @@ export class DependencyUtils {
         }
     }
 
+    private static getUsageFeaturesByExistTech(projectDescriptors: Map<PackageType, vscode.Uri[]>, scanSuffix: string): IUsageFeature[] {
```

Comment:
I don't think this is the best place to have this utils inside `dependencyUtils.ts`

Reply:
moved to `scanUtils.ts`